shell/linters/kubecfg.sh

```diff
@@ -30,7 +30,7 @@ kubecfg_kubeconform() {
     -schema-location default \
     -ignore-missing-schemas \
     -strict \
-    -kubernetes-version 1.24.15 \
+    -kubernetes-version 1.25.16 \
```

Comment:
suggestion: Perhaps this version should be declared in `versions.yaml` and then loaded from that file in here?

Reply:
sure